### PR TITLE
database-performance-audit

### PR DESCRIPTION
**Pull request by Patfarmurs**
Function
>- improve the speed for each of the queries.

Screenshot1-day5

Screenshot2-day5

Screenshot3-day5

Screenshot4-day5

Screenshot5-day5

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -188,4 +188,19 @@ SELECT vets.name, species.name, count(species.name) from vets
...
+
+SET work_mem = '128MB';
+
+ANALYZE visits;
+
+CREATE INDEX idx_email ON owners (email);
+
+ANALYZE owners;
+
+CREATE INDEX idx_vet ON visits (vet_id);
```
Comment (mikethreels) [inline queries.sql]: - [x] all the CREATE queries should be put in the `schema.sql` file

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Teams, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good job improving the speed.

## Required Changes ♻️
- [x] please make sure to include print screens of all the queries before and after improvement.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (touredri) [reply queries.sql]: ok

**Review by christianbiring1 — APPROVED** [review]
Hi @Patfarmurs , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights
- [ ] Great on implementing the changes as required by the previous reviewer :+1: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Patfarmurs** [comment]
Thanks @christianbiring1

**Pull request merged**